test/connectivity/NetworkController/LinuxFirewallCommands.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace NetworkController
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    class LinuxFirewallCommands : IFirewallCommands
+    {
+        static readonly ILogger Log = Logger.Factory.CreateLogger<LinuxFirewallCommands>();
+        readonly string networkInterfaceName;
+
+        public LinuxFirewallCommands(string networkInterfaceName)
+        {
+            this.networkInterfaceName = networkInterfaceName;
+        }
+
+        public async Task<bool> SetAcceptRule(CancellationToken cs)
+        {
+            var exitCode = await CommandExecutor.Execute(
+                "tc",
+                $"qdisc delete dev {this.networkInterfaceName} root netem loss 100%",
+                cs);
+
+            var exitMessage = exitCode == 0 ? "Removed" : $"Exit code {exitCode} failed to remove";
+            Log.LogInformation($"{exitMessage} tc loss rule for {this.networkInterfaceName}");
+
+            return exitCode == 0;
+        }
+
+        public async Task<bool> SetDropRule(CancellationToken cs)
```

Comment:
is this method used to drop all network packets of given network interface? if so, what is the difference between drop all packets and IOfflineController?